src/containers/DataDisplay/DataViewTable.tsx

```diff
@@ -52,30 +51,13 @@ const DataViewTable: React.FC = () => {
     filtersReady: !filtersLoading,
   })
 
-  const [currentSortColumn, setCurrentSortColumn] = useState<SortColumn>(null)
-
   const title = location?.state?.title ?? dataViewTable?.title ?? ''
   usePageTitle(title)
 
-  // Reset filters if navigating here from Menu bar
-  useEffect(() => {
-    if (location?.state?.resetFilters) {
-      setGqlFilter({})
-      setApiQueries({})
-      setSearchText('')
-      setDebounceInput('')
-    }
-  }, [dataViewCode])
```

Comment:
This resetting wasn't quite capturing all cases (it still left "activeFilters", even though they had nothing in them), so moved to "ListFilters" instead. Works better there anyway, as just resetting "activeFilters" resets all of the above as a result.